aws/request.go

```diff
@@ -389,19 +383,6 @@ func (r *Request) getNextRequestBody() (body io.ReadCloser, err error) {
 			"failed to get request body error", err)
 	}
 
-	// Go 1.8 tightened and clarified the rules code needs to use when building
```

Comment:
I think we want to keep this doc comment. The SDK still needs to not set the `http.Request.Body` value when there has not been a body specified by the SDK's request. WE should have a github issue tracking this fix.